obs/project.go

```diff
@@ -151,6 +153,61 @@ func (c *Client) CreateUpdateProject(ctx context.Context, project *Project) erro
 	return nil
 }
 
+// GetProjectMetaFile is used to get the contents of the obs project meta file
+func (c *Client) GetProjectMetaFile(ctx context.Context, project *Project) ([]byte, error) {
```

Comment:
There are multiple problems here:
- This function shouldn't take the Project struct, it should only take the project's name
- This function shouldn't return byte slice, instead it should return a pointer to a Project struct
  - In this case, you need to unmarshal the response into a Project struct at the end
  - However, this might not be scalable because there might some fields in the meta that we don't have in our struct. But let's ignore this for now and just unmarshal the response into the Project struct that we already have

Reply:
Resolved